noodles-editor/src/noodles/operators.ts

```diff
@@ -1719,6 +1719,157 @@ export class DirectionsOp extends Operator<DirectionsOp> {
   }
 }
 
+export class OverpassOp extends Operator<OverpassOp> {
+  static displayName = 'Overpass'
+  static description = 'Query OpenStreetMap data using Overpass API'
+  asDownload = () => this.outputData
+  createInputs() {
+    return {
+      query: new CodeField(
+        '[out:json][timeout:25];\n(\n  node["name"="Central Park"];\n  way["name"="Central Park"];\n);\nout geom;',
+        {
+          language: 'overpass-ql',
+        }
+      ),
+      // TODO: We should add a bbox type
+      bbox: new UnknownField([{ lng: -74.006, lat: 40.7128 }, { lng: -73.935, lat: 40.73061 }], { optional: true }),
+      endpoint: new StringField('https://overpass-api.de/api/interpreter'),
+      pulse: new NumberField(0, { min: 0, step: 1 }),
+    }
+  }
+  createOutputs() {
+    return {
+      data: new DataField(),
+    }
+  }
+  async execute({
+    query,
+    bbox,
+    endpoint,
+  }: ExtractProps<typeof this.inputs>): Promise<ExtractProps<typeof this.outputs>> {
+    try {
+      // Replace {{bbox}} template with actual coordinates if bbox is provided and valid
```

Comment:
I don't love that their syntax is similar to ours, but semantically this refers to a param on this operator instead of a global reference.

Trying to figure out the right balance here...